test/utils/int/common.ts

```diff
@@ -75,3 +78,32 @@ export function setupClosedCenterBookends(): void {
         }
     });
 }
+
+/**
+ * Generates an object with the shape that matches a notification in the database.
+ */
+export function generateStoredNotification(source: Identity, notificationOptions?: Partial<NotificationInternal>): StoredNotification {
+    const id = notificationOptions && notificationOptions.id || `${source.uuid}:${Math.random()*Math.random()}`;
```

Comment:
I don't think Math.random()*Math.random() gets us anything aside from a non-uniform distribution?

Reply:
This method is no longer needed so I have removed it altogether